public/app/features/plugins/extensions/getPluginExtensions.ts

```diff
@@ -192,3 +203,21 @@ function getLinkExtensionOnClick(
     }
   };
 }
+
+function getLinkExtensionPathWithTracking(
+  pluginId: string,
+  path: string | undefined,
```

Comment:
nit: accept always a string, let the caller of this method handle the correct conditions to pass a correct value instead of falling back here.